tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -286,6 +292,35 @@ public function addProductsToOrderWithExistingInvoiceAndTheFollowingDetails(stri
         }
     }
 
+    /**
+     * @Then :invoicePosition invoice from order :orderReference should have following details:
+     *
+     * @param string $invoicePosition
+     * @param string $orderReference
+     * @param TableNode $table
+     */
+    public function checkInvoiceDetails(string $invoicePosition, string $orderReference, TableNode $table)
+    {
+        $orderId = SharedStorage::getStorage()->get($orderReference);
```

Comment:
```suggestion
        $orderId = (int) SharedStorage::getStorage()->get($orderReference);
```

Reply:
Not needed because the stored order id is always a strict int